monitoring/base/prometheus/alert_rules/ingress-watcher.yaml

```
@@ -0,0 +1,21 @@
+groups:
+  - name: ingress-watcher
+    rules:
+      - alert: IngressDown
+        expr: |
+          rate(ingresswatcher_http_get_successful_total{code="200 OK"}[5m]) / on(job,path) rate(ingresswatcher_http_get_total[5m]) < 0.95
+        labels:
+          severity: critical
+        for: 10m
+        annotations:
+          summary: ingress {{ $labels.path }} is down (from {{ $labels.job }}).
+          runbook: TBD
+      - alert: IngressWatcherDown
+        expr: |
+          sum by(job)(rate(ingresswatcher_http_get_total[5m])) / count without(path)(rate(ingresswatcher_http_get_total[5m])) * ingresswatcher_watch_interval < 0.95
```

It is better to use `without`.
```suggestion
          sum without(path)(rate(ingresswatcher_http_get_total[5m])) / count without(path)(rate(ingresswatcher_http_get_total[5m])) * ingresswatcher_watch_interval < 0.95
```

---

Maybe, `avg` could be used.
```suggestion
          avg without(path)(rate(ingresswatcher_http_get_total[5m]))  * ingresswatcher_watch_interval < 0.95
```